app/views/user/_owner_profile.html.slim

```diff
@@ -39,11 +39,15 @@ section.user-profile
           br
             
 .columns.list
-  article.maincol
-    =render '/dashboard/collection_index'
+  article.maincol  
+    -if any_public_collections_with_document_sets?(collections_and_document_sets)
+      =render partial: '/dashboard/hierarchical_collections_and_document_sets', locals: {collections_and_document_sets: collections_and_document_sets}
+    -else
+      =render partial: '/dashboard/alphabetical_collections_and_document_sets', locals: {collections_and_document_sets: collections_and_document_sets}
+    
```

Comment:
This is the main logic that decides if we use the hierarchical or alphabetical look.